mistralrs-core/src/sequence.rs

```diff
@@ -7,6 +7,7 @@ use std::{
 
 use candle_core::Tensor;
 use candle_sampling::logits_processor::{LogitsProcessor, Logprobs};
```

Comment:
I recently moved the sampler into `mistralrs-core`. Could you do a git merge to update?